### PR TITLE
[stable/traefik] Add configurable PVC annotations

### PR DESCRIPTION
**Pull request by siwyd**
When migrating one traefik setup to another, it's handy to be able
set annotations on the PVC, especially "helm.sh/resource-policy".

### CODE REVIEW

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.24.1
+version: 1.25.0
```
Comment (krancour) [inline stable/traefik/Chart.yaml]: [Something else](https://github.com/kubernetes/charts/pull/3950) got in ahead of this. This Is going to have to b1 `1.26.0`. After that, LGTM and we'll get it merged.

Comment (siwyd) [reply stable/traefik/Chart.yaml]: Alright, will change and rebase!

**Comment by siwyd** [comment]
@krancour Chart version bumped and rebased.

**Pull request merged**